server/test/unit/apiTest/authorizeTest.js

```diff
@@ -6,28 +6,16 @@ const sinonChai = require('sinon-chai')
 
 // In-house dependencies
 const { helpers } = require('brave-alert-lib')
-
+const { mockResponse } = require('../../../testingHelpers')
 const api = require('../../../api')
 
 // Configure Chai
 use(sinonChai)
 
 const sandbox = sinon.createSandbox()
 
-function authorize_next() {
-  helpers.log('authorize_next')
-}
```

Comment:
Just for my own understanding - could you explain why you removed this `authorize_next` function and replace it with `authorizezNext` ? Was it just to use camelCase?

Reply:
Exactly this! The `authorize_next` function here was from an older PR made by myself, and it wasn't caught... Theresa had previously made comments that all variables, functions, etc., in the JS repositories should consistently use camelCase, and I am fixing this mistake for that reason. :)

Reply:
Got it! Thanks for explaining!